site/en/docs/web-platform/compute-pressure/index.md

```diff
@@ -1,264 +1,221 @@
-
-The same threshold and bucket system applies to the exposed CPU speed values.
-
-{% Aside %}
-The values reported by the API reflect the overall state of the CPU,
-due to utilization across all applications currently running. Therefore, even if
-your own web application is almost entirely idle, you may see high utilization
-and clock speed values reported, due to the activities of other applications.
-{% endAside %}
-
-## Try a demo
-
-When accessed on a browser that supports the origin trial (currently Chrome 92
-or later), [this demo](https://googlechrome.github.io/samples/compute-pressure/index.html)
-will display an ongoing readout of your device's CPU utilization and speed.
-
-
-## Acknowledgements
-
-This blog post is derived on the sample code and
-[API explainer](https://github.com/oyiptong/compute-pressure/blob/main/README.md)
-created by Olivier Yiptong and Victor Costan.
-
-The hero image was created by Robert Anasch on
-[Unsplash](https://unsplash.com/photos/-C7IKRBZHrg). The diagram illustrating
-bucketing was created by [Adam Argyle](/authors/adamargyle/).
+In this example the `cpu` is the pressure source we are interested in. For now, it is the only
+source available. In the future, there may be other sources such as `gpu`, `power`.
+
+To stop observing a source, use the `unobserve()` method, as in the following example:
+
+```js
+observer.unobserve('cpu');
+```
+
+In order to unobserve all sources at once, use the `disconnect()` method, as in the following example:
+
+```js
+observer.disconnect();
+```
+### Retrieving pressure records
+The `takeRecords()` method of the `PressureObserver` interface returns an array of
+`PressureRecords' objects stored in the pressure observer, emptying it out.
+
+Calling this method clears the pending records list, so the callback will not be run.
+
+```js
+function callback(records) {
+  const lastRecord = records[records.length - 1];
+  console.log(`Current pressure ${lastRecords.state}`);
+}
+
+const observer = new PressureObserver(callback, { sampleRate: 1 });
+await observer.observe('cpu');
+
+// forced record reading
+const records = observer.takeRecords();
+```
+### Tell us about the API design
+Is there anything about the API that does not work as you expected? Do you see any missing method
+or property for your usage of the API? File a spec issue or comment on an existing one in the
+corresponding [GitHub repo][issues].
+
+### Report a problem with the implementation
+Did you find a bug with Chromium's implementation? Or is the implementation different from the
+spec?
+File a bug at [new.crbug.com](https://new.crbug.com). Be sure to include as much detail as you can, simple instructions
+for reproducing, and enter [Blink>PerformanceAPIs>ComputePressure][blink-component] in the **Components** box.
+
+## Helpful links {: #helpful }
+- [Specifications] [specs]
+- [Public explainer][explainer]
+- [Compute Pressure API Demo][demo] | [Compute Pressure API Demo source][demo-source]
```

Comment:
Demo crashes chrome tab in Canary with experimental web platform features flag enabled. Is this expected?
![image](https://user-images.githubusercontent.com/634478/229499053-2290d670-d207-4eba-8beb-e622cea5b380.png)

Reply:
Note that it only happens when I don't run Chrome with the command line flag `--enable-features=ComputePressure`.

Reply:
ha yes, maybe we could have a more graceful way to exit, if feature is not enabled.